applications/openshift/api-server/api_server_api_priority_v1alpha1_flowschema_catch_all/tests/ocp4/4.10.yml

```diff
@@ -1,2 +1,3 @@
 ---
 default_result: NOT-APPLICABLE
+
```

Comment:
Is the test result here saying this passed?

https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/ComplianceAsCode_content/9023/pull-ci-ComplianceAsCode-content-master-e2e-aws-ocp4-high/1539958508517068800/artifacts/e2e-aws-ocp4-high/test/build-log.txt

```
   helpers.go:807: Result - Name: e2e-high-api-server-api-priority-flowschema-catch-all - Status: PASS - Severity: medium
```

Reply:
Good catch, but the flowcontrol checks should be fixed in a separate PR (currently under test), see CMP-1416, WIP  is at https://github.com/jhrozek/content/commits/flowschema_versions I just didn't manage to finish the work yet